scripts/keras_benchmarks/models/mnist_irnn_benchmark.py

```diff
@@ -0,0 +1,112 @@
+'''
+Original Model from keras/examples
+
+This is a reproduction of the IRNN experiment
```

Comment:
Similarly, I think this script wouldn't be a good fit for a benchmark, since we don't need to benchmark `SimpleRNN`. I think it would be good to have a benchmark with GRU and one with a stack of LSTM layers. This will allow us to test different implementations (in particular, the cuDNN layers).

Please remove this one. We'll re-introduce new benchmarks later (in a new PR).